test/test_transform.py

```diff
@@ -0,0 +1,114 @@
+from pseudospectral import DiracOperator, Derivative1D
```

Comment:
We don't need an operator here. This is a property of the spectrum.
```suggestion
from pseudospectral import Derivative1D
```
and write all the tests using the spectrum itself.

Reply:
Done